mappings/net/minecraft/client/gui/screen/world/SelectWorldScreen.mapping

```diff
@@ -18,7 +18,7 @@ CLASS net/minecraft/unmapped/C_yrypplpi net/minecraft/client/gui/screen/world/Se
 		ARG 1 button
 	METHOD m_qjteouzv (Lnet/minecraft/unmapped/C_buwziidm;)V
 		ARG 1 button
-	METHOD m_rkjktgal worldSelected (ZZ)V
+	METHOD m_rkjktgal updateButtonActivationStates (ZZ)V
```

Comment:
This matches `MultiplayerScreen.updateButtonActivationStates`, which does the same thing.